autoload/ale/lsp/tsserver_message.vim

```diff
@@ -37,11 +37,14 @@ function! ale#lsp#tsserver_message#Geterr(buffer) abort
 endfunction
 
 function! ale#lsp#tsserver_message#Completions(buffer, line, column, prefix) abort
+    let l:external = get(g:, 'ale_completion_tsserver_autoimport', 1)
```

Comment:
Turn this off by default instead. That what current behaviour won't change, and we'll continue to show more refined results. Compute the option in the `completion.vim` file instead. These functions should not check any settings, they are just for getting the messages.